docs/timezone.md

```diff
@@ -8,13 +8,68 @@
 A `Temporal.TimeZone` is a representation of a time zone:
 
 - either an [IANA time zone](https://www.iana.org/time-zones), including information about the time zone, such as the offset between the local time and UTC at a particular time, daylight saving time (DST) and other political UTC offset changes like a country having permanently changed its offset;
-- or simply a particular UTC offset with no offset transitions.
+- or an "offset time zone": a fixed UTC offset with no offset transitions.
 
-Since `Temporal.Instant` and `Temporal.PlainDateTime` do not contain any time zone information, a `Temporal.TimeZone` object is required to convert between the two.
+To combine a time zone with a date/time value, and to perform DST-safe operations like "add one day", use [`Temporal.ZonedDateTime`](./zoneddatetime.md).
+
+## Time zone identifiers
+
+Time zones in `Temporal` are represented by string identifiers from the IANA Time Zone Database (like `Asia/Tokyo`, `America/Los_Angeles`, or `UTC`) or a fixed UTC offset like `+05:30`.
+Each time zone has a canonical identifier, and may also have one or more aliases.
+
+Anywhere in ECMAScript that a `Temporal.TimeZone` object is expected, a time zone identifier can be used instead.
+Furthermore, using string identifiers instead of time zone objects allows ECMAScript implementations to perform optimizations that are not possible when using time zone objects.
+Therefore, it's recommended to always use string identifiers instead of time zone objects whenever this is convenient. For example:
+
+```javascript
+inBerlin = Temporal.ZonedDateTime.from('2022-01-28T19:53+01:00[Europe/Berlin]');
+inTokyo = inBerlin.withTimeZone('Asia/Tokyo'); // May be faster / use less RAM
+inTokyo = inBerlin.withTimeZone(Temporal.TimeZone.from('Asia/Tokyo')); // OK, but not optimal
+```
+
+### Canonicalization of time zone identifiers
+
+The `id` property of any `Temporal.TimeZone` instance returns the canonical identifier for that time zone, even if the input used to create that instance used an alias and/or a different mix of upper and lower case letters. For example:
+
+```javascript
+// In Firefox 109, India's time zone's canonical identifier is 'Asia/Kolkata'
+Temporal.TimeZone.from('Asia/Kolkata').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('Asia/Calcutta').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('asia/calcutta').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('ASIA/CALCUTTA').id; // => 'Asia/Kolkata'
+
+// In Chrome 109, India's time zone's canonical identifier is 'Asia/Calcutta'
+Temporal.TimeZone.from('Asia/Kolkata').id; // => 'Asia/Calcutta'
+Temporal.TimeZone.from('Asia/Calcutta').id; // => 'Asia/Calcutta'
```

Comment:
Let's not call extra attention to this discrepancy between implementations — I'm hoping it'll be solved at some point and then this will be outdated. (I still think it's a Chrome bug.) Could we choose a different one where implementations agree with each other?